tensorflow/examples/udacity/1_notmnist.ipynb

```diff
@@ -496,22 +502,61 @@
         "outputId": "8af66da6-902d-4719-bedc-7c9fb7ae7948"
       },
       "source": [
+        "def make_arrays(nb_rows, img_size):\n",
+        "    if nb_rows:\n",
+        "        dataset = np.ndarray((nb_rows, img_size, img_size), dtype=np.float32)\n",
+        "        labels = np.ndarray(nb_rows, dtype=np.int32)\n",
+        "    else:\n",
+        "        dataset, labels = None, None\n",
+        "    return dataset, labels\n",
+        "\n",
+        "def merge_datasets(pickle_files, train_size, valid_size=0):\n",
+        "    num_classes = len(pickle_files)\n",
+        "    valid_dataset, valid_labels = make_arrays(valid_size, image_size)\n",
+        "    train_dataset, train_labels = make_arrays(train_size, image_size)\n",
+        "    vsize_per_class = int(valid_size / num_classes)\n",
```

Comment:
The call to `int()` make it Python3-compatible, but you're right it is clearer with //.